pkg/virt-api/validating-webhook/validating-webhook_test.go

```diff
@@ -771,15 +776,42 @@ var _ = Describe("Validating Webhook", func() {
 
 			causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vm.Spec)
 			Expect(len(causes)).To(Equal(1))
-			Expect(causes[0].Field).To(Equal("fake.domain.devices.interfaces[0].ports[1]"))
+			Expect(causes[0].Field).To(Equal("fake.domain.devices.interfaces[0].ports[0]"))
 		})
-		It("should reject networks with a pod network source and slirp interface with two same Ports and Protocol", func() {
+		It("should reject interface with two port with the same name", func() {
 			vm := v1.NewMinimalVMI("testvm")
 			vm.Spec.Domain.Devices.Interfaces = []v1.Interface{v1.Interface{
 				Name: "default",
 				InterfaceBindingMethod: v1.InterfaceBindingMethod{
-					Slirp: &v1.InterfaceSlirp{Ports: []v1.Port{{Port: 80}, {Protocol: "UDP", Port: 80}, {Protocol: "TCP", Port: 80}}},
```

Comment:
You seem to have lost a test case for when there are three ports, and first and last are duplicates.

Reply:
We stick with the kubernetes configuration.
In kubernetes you can configure the same port multiple time, the only thing that need to be unique is the name (if you gave one)